GaugeKit/Gauge+Animations.swift

```diff
@@ -9,15 +9,15 @@
 import UIKit
 
 extension Gauge {
-    public func animateRate(duration: NSTimeInterval, newValue: CGFloat, completion: (Bool) -> ()) -> Void {
+    public func animateRate(_ duration: TimeInterval, newValue: CGFloat, completion: @escaping (Bool) -> ()) -> Void {
```

Comment:
You are keeping a reference to `completion` on `animationCompletionBlock` later on.